*** domain.te	2016-07-15 09:09:02.423054200 -0700
--- domain-Copy.te	2016-07-15 09:10:23.510797900 -0700
***************
*** 297,311 ****
      userdebug_or_eng(`-su')
      -system_server
      -zygote
- } { file_type -system_file -exec_type }:file execute;
- neverallow {
-     domain
      -appdomain # for oemfs
      -recovery # for /tmp/update_binary in tmpfs
  } { fs_type -rootfs }:file execute;
  
  # Only the init property service should write to /data/property.
! neverallow { domain -init -recovery } property_data_file:dir no_w_dir_perms;
  neverallow { domain -init -recovery } property_data_file:file no_w_file_perms;
  
  # Only recovery should be doing writes to /system
--- 297,308 ----
      userdebug_or_eng(`-su')
      -system_server
      -zygote
      -appdomain # for oemfs
      -recovery # for /tmp/update_binary in tmpfs
  } { fs_type -rootfs }:file execute;
  
  # Only the init property service should write to /data/property.
! #neverallow { domain -init -recovery } property_data_file:dir no_w_dir_perms;
  neverallow { domain -init -recovery } property_data_file:file no_w_file_perms;
  
  # Only recovery should be doing writes to /system
